kinto_http/session.py

```diff
@@ -5,6 +5,7 @@
 
 import requests
 
+import kinto_http
 from kinto_http import utils
```

Comment:
Any reason in particular you chose not to import `BearerTokenAuth` from `kinto_http` when we're already importing utils? Not a big deal, just curious

Reply:
I didn't really think about it. I had `kinto_http.BearerTokenAuth` in mind and imported the top module to get it...

Reply:
Ah well, my subconscious was probably aware that it would raise:
```
ImportError: cannot import name 'BearerTokenAuth' from partially initialized module 'kinto_http' (most likely due to a circular import) (/Users/mathieu/Code/Mozilla/kinto-http.py/kinto_http/__init__.py)
```

🤣